src/contexts/EtherspotContext.tsx

```diff
@@ -18,13 +19,17 @@ export interface EtherspotContextData {
     getSdkForChainId: (chainId: number, forceNewInstance?: boolean) => EtherspotSdk | null;
     isConnecting: boolean;
     sdk: EtherspotSdk | null;
+    balancePerChainSmartWallet: IBalanceByChain[] | null;
```

Comment:
naming convention feels wrong, maybe change it to `smartWalletBalancePerChain`?

Reply:
Okh will use `smartWalletBalanceByChain` for all